### DISCUSSION TITLE
Mockups for the UI (Upvote your favorites & Post yours)

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#4] [Development]
^^ See Title

**Comment by duubi-official** [comment]
Hi! I'd love to help with the UI. But first what should it have?

**Reply by icarusinthewings** [reply]
We need a Homepage, Collections (Studios), Projects, and User pages. I'm working on a logo.

**Comment by duubi-official** [comment]
How does this font look? https://tokotype.github.io/plusjakarta-sans/

**Reply by icarusinthewings** [reply]
That's good. I will implement this into the UI.

**Comment by MikeDev101** [comment]
Need a mockup layout? I'll see what I can come up with.

_photoshop and illustrator finally getting some use for once lol_

**Reply by icarusinthewings** [reply]
Haha. Ever since I moved to linux, I've been using Inkscape and Gravit.

**Reply by MikeDev101** [reply]
Here we go! I tried to make the little transparent-gradient stars on the side but gave up.

![polaris_concept_art_front_page](https://user-images.githubusercontent.com/12957745/125148509-c073db80-e100-11eb-94ba-55fb72e48f78.png)

**Reply by MikeDev101** [reply]
I lazily came up with a design, sorry about it being a bit sloppy. It's been a while since I've used Illustrator...

**Reply by MikeDev101** [reply]
edit: saw the repl.it preview. Wow. Did not expect it to look like that, I'll redesign the concept art to fit better with it.

**Reply by icarusinthewings** [reply]
Awesome!
Can you make the buttons into symbols for the Create, Explore, and Login?
Also:
The site will allow you to turn off remixing & see inside, along with allowing custom licenses including (C), CC0, MIT, GPL3, and LGPL

**Reply by MikeDev101** [reply]
> Awesome!
> Can you make the buttons into symbols for the Create, Explore, and Login?
> Also:
> The site will allow you to turn off remixing & see inside, along with allowing custom licenses including (C), CC0, MIT, GPL3, and LGPL

Support for any license would be nice. I'll see what I can do with that.

**Comment by dannyaguero** [comment]
Here are some designs!
NOTE: Since these are just concepts (and I ain't no Picasso), The concepts are simple.
<img width="1309" alt="Screen Shot 2021-08-04 at 8 11 13 PM" src="https://user-images.githubusercontent.com/52724381/128271461-1ba420a8-3691-4307-83ee-1c7f3cc75798.png">
<img width="1309" alt="Screen Shot 2021-08-04 at 8 11 31 PM" src="https://user-images.githubusercontent.com/52724381/128271473-0e1c4909-0f27-4aeb-82fc-327f7de3fa85.png">
<img width="1309" alt="Screen Shot 2021-08-04 at 8 11 42 PM" src="https://user-images.githubusercontent.com/52724381/128271479-4192fb8f-4bae-4aa7-af65-b28d08c682a9.png">
<img width="1309" alt="Screen Shot 2021-08-04 at 8 11 50 PM" src="https://user-images.githubusercontent.com/52724381/128271488-b0fad516-0b53-431a-93b8-205ded93117d.png">
Hopefully they are good! Bye now.

**Reply by MikeDev101** [reply]
Not a bad layout! However, I feel like we should also have a layout of how a project sharing UI would look like (Name your project, provide instructions, select a license, and share)

**Reply by CloudGate1234** [reply]
bit of an overload. It'd be better if it were simpler. I like the scratch layout to be honest. The project is big. the title medium. everything else, small. Scrolling down to reach the comments is easy. 

The homepage would be good If it looked somewhat like this https://www.asurascans.com/. Your mockup the "see more button looks too big. I don't see why you would need your follow list on the homepage rather than on your profile. 

The profile page looks ok, but I think this layout is best except with only one description with a bigger description box
![image](https://user-images.githubusercontent.com/88938765/129451279-3a57a264-37f7-4e7b-a440-a1a0b0128996.png)

The studio is good, but the see more should be at the bottom.


This is just my opinion

**Reply by MikeDev101** [reply]
I was thinking of a more vertical layout, one that would work better with smaller screens or weirder aspect ratios.

![polaris thing](https://user-images.githubusercontent.com/12957745/129466023-46ba915a-144e-4bb9-8684-c041e0170eae.png)

**Reply by CloudGate1234** [reply]
> I was thinking of a more vertical layout, one that would work better with smaller screens or weirder aspect ratios.
> 
> ![polaris thing](https://user-images.githubusercontent.com/12957745/129466023-46ba915a-144e-4bb9-8684-c041e0170eae.png)
MAybe that for mobile, but I think a scratch-like website would be good for pc
